JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +34,34 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        static cookieSeparator: string = '|';
+
+        public setAuthId(id: string, accountId? : string) {
+            this.authId = id;
+            var authCookie = this.authId;
+
+            if (accountId) {
+                this.accountId = accountId;
+                authCookie = [this.authId, this.accountId].join(User.cookieSeparator);
+            }
+            
+            // Set the suth id cookie. No expiration date because this is a session cookie (expires when browser closed).
+            Util.setCookie('ai_authUser', authCookie);
+        }
+
+        public clearAuthId() {
+            this.authId = null;
+            this.accountId = null;
```

Comment:
accountid is not only for auth id, can you double check that you're supposed to delete it when you clear auth id?